raven-lua-scm-1.rockspec

```diff
@@ -15,6 +15,7 @@ to Sentry.]],
 dependencies = {
   "lua >= 5.1",
   "lua-cjson",
+  "lua-resty-http"
```

Comment:
I usually not add senders dependencies on the rockspec because it doesn't make sense to every user of the module to install them. This is why `luasocket` and `luasec` are not listed here as they are specific to the `luasocket` backend.